pkg/template/registry/templateinstance/strategy.go

```diff
@@ -118,23 +119,30 @@ func SelectableFields(obj *templateapi.TemplateInstance) fields.Set {
 	return templateapi.TemplateInstanceToSelectableFields(obj)
 }
 
+func (s *templateInstanceStrategy) authorize(u user.Info, action *authorizationapi.Action) error {
```

Comment:
Yes - but I don't know where - is somewhere in pkg/authorization that this could/should go?  I'm sort of surprised there isn't one already.